haskell/package.yaml

```diff
@@ -4,7 +4,7 @@
 <<: !include "./hpack/module.yaml"
 
 name:                stablecoin
-version:             1.3.0
+version:             1.4.0
```

Comment:
Please also update the changelog:
1. Record your changes there.
2. Update version to 1.4.0.